content/reports/1000_IBM Power Virtual Machines - Reports/010_PVMs_ Hardware.yaml

```diff
@@ -0,0 +1,77 @@
+---
+dims:
+created_on: 2021-07-28 18:04:00.000000 Z
+title: "Detailed Installed PowerVC and PowerVS VM Report"
+conditions: !ruby/object:MiqExpression
```

Comment:
@agrare @Fryguy @kbrock recalling https://github.com/ManageIQ/manageiq/pull/20907, should this be a string that gets interpreted as a regular expression vs an `MiqExpression` object?

Reply:
A. It would be great if `MiqReport.seed` handled converting a hash -> MiqExpression object (I don't know reporting well enough to know if there are other possible data types for `:conditions`)

B. I don't think it is feasible to do that for this PR, this is used extensively in our existing reports in core.  Also this is a different situation from https://github.com/ManageIQ/manageiq/pull/20907 because the API doesn't have to serialize the report to JSON then post the JSON back to the original format.

Reply:
a. lets not change this here
b. I really like the suggested api update
c. https://github.com/ManageIQ/manageiq/pull/21365 allows us to move forward in the future if we desire